docker/Dockerfile

```diff
@@ -12,22 +12,26 @@ RUN make binary-frontend
 
 
 ######## Build the backend
-FROM golang:1.20-bullseye as backendbuild
+FROM golang:1.20-bookworm as backendbuild
 
 WORKDIR /go/src/github.com/analogj/scrutiny
 COPY --link . /go/src/github.com/analogj/scrutiny
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive \
+    apt-get install -y --no-install-recommends \
+    file
 RUN make binary-clean binary-all WEB_BINARY_NAME=scrutiny
 
 
 ######## Combine build artifacts in runtime image
-FROM debian:bullseye-slim as runtime
+FROM debian:bookworm-slim as runtime
 ARG TARGETARCH
 EXPOSE 8080
 WORKDIR /opt/scrutiny
 ENV PATH="/opt/scrutiny/bin:${PATH}"
 ENV INFLUXD_CONFIG_PATH=/opt/scrutiny/influxdb
 ENV S6VER="1.21.8.0"
 ENV INFLUXVER="2.2.0"
+SHELL ["/usr/bin/sh", "-c"]
```

Comment:
Is this necessary?

Reply:
Apparently so. When I first tried to rebase to bookworm, the build would generate errors about missing /bin/sh. According to https://www.debian.org/releases/bookworm/amd64/release-notes/ch-information.en.html#a-merged-usr-is-now-required, /bin no longer holds common comnands like /bin/sh (although they should still work). Changing the shell in the docker file seems to solve the issue.